### ISSUE TITLE
[es]: Language Request Spanish

### ISSUE THREAD

**Issue by darigovresearch** [darigovresearch/Intro-to-DIY-Off-Grid-Systems#5]
Spanish has been requested, having the following in Spanish to build out the folder structure would be helpful to begin the translation of each chapter

1. Preface
2. Introduction
3. Basic concepts
4. What is an off grid system
5. Generation
6. Storage
7. Charge control
8. Discharge control
9. Other system level equipment
10. Usage level equipment
11. Appliances
12. Constructing a Simple 12V Off-Grid Circuit
13. Basic system modelling
14. Designing an efficient system
15. Planning Your System
16. Calculating your consumption
17. Calculating battery storage
18. Calculating solar generation
19. System losses
20. Additional system modelling
21. Constructing the Whole System
22. Resources
23. Appendices

Other terms
1. Intro to DIY Off Grid Systems

**Comment by sandaluz-aptera** [comment]
Is this still available? I can help with translation in Spanish

**Comment by darigovresearch** [comment]
Yeah feel free to comment below the above items which need to be translated so we can auto generate the file structure to make it easier for you to translate the rest of it.

You can also take a look at the other issues on other repositories if you like this topic and want to translate more

https://github.com/darigovresearch/DIY-Bike-Power/issues/2

https://github.com/darigovresearch/DIY-Solar-Phone-Chargers/issues/3

https://github.com/darigovresearch/Make-Your-Own-Lithium-Power-Banks/issues/16
